apps/email/js/cards/compose.js

```diff
@@ -749,6 +742,37 @@ return [
       else {
         this.attachmentsContainer.classList.add('collapsed');
       }
+      this.updateAttachmentsAriaLabel();
+    },
+
+    /**
+     * Update the label used for accessibility that describes the attachments
+     * included.
+     */
+    updateAttachmentsAriaLabel: function() {
+      if (this.composer.attachments.length <= 1) {
+        mozL10n.setAttributes(this.attachmentsContainer,
+          'compose-attachments-container');
+        return;
+      }
+      var kilobytes = Math.ceil(this.calculateTotalAttachmentsSize() / 1024);
+      mozL10n.setAttributes(this.attachmentsContainer,
+        'compose-attachments-container-kib', { kilobytes: kilobytes });
```

Comment:
As discussed in IRC, let's see if just compose-attachments-container-kib key is enough here and use the plural form in the .properties file, so that we do not need the compose-attachments-container option, and pass 0 for no attachments case and use the [zero] .properties option to get the same effect.